# Reinstate the empty -O option to fix weird mixing of native and target
# binaries and libraries with LD_LIBRARY_PATH when host==target
#
# Signed-off-by: Denys Dmytriyenko <denis@denix.org>

Index: Python-2.6.1/Modules/main.c
===================================================================
--- Python-2.6.1-orig/Modules/main.c
+++ Python-2.6.1/Modules/main.c
@@ -352,6 +352,9 @@ Py_Main(int argc, char **argv)
 
 		/* case 'J': reserved for Jython */
 
+		case 'O': /* ignore it */
+			break;
+
 		case 'N':
 			Py_OptimizeFlag=0;
 			break;
